clients/java/src/main/java/io/camunda/client/impl/response/DocumentReferenceResponseImpl.java

```diff
@@ -53,4 +69,50 @@ public String getContentHash() {
   public DocumentMetadata getMetadata() {
     return metadata;
   }
+
+  public io.camunda.client.protocol.rest.DocumentReference getDocumentReference() {
```

Comment:
```suggestion
  public DocumentReference getDocumentReference() {
```

Reply:
I am not sure why I did not use import here.